packages/features/form-builder/FormBuilder.tsx

```diff
@@ -288,6 +288,13 @@ export const FormBuilder = function FormBuilder({
               showToast(t("form_builder_field_already_exists"), "error");
               return;
             }
+            //handling edge-case when user clears the min max length values.
+            if (!data?.minLength) {
+              delete data.minLength;
+            }
+            if (!data?.maxLength) {
+              delete data.maxLength;
+            }
```

Comment:
Why do we need to delete the props? Using delete operator doesn't seem like a good choice to solve  this problem.

Reply:
> Why do we need to delete the props? Using delete operator doesn't seem like a good choice to solve  this problem.

What do you suggest?. When user clears the value and save the field it gets stored as NaN.